### PR TITLE
03/jdix

### PR DESCRIPTION
**Pull request by jdix531**
# Requirements for Contributing a Bug Fix

- Fill out the template below. Any pull request that does not include enough
  information to be reviewed in a timely manner may be closed at the maintainers'
  discretion.

### CODE REVIEW

File: 03-iam/lab-3.3.2/cfn.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          - Effect: Allow
+            Action: "s3:*"
+            # Resource: '*'
+            Resource: !Join [ '', [ !GetAtt MyFirstBucket.Arn, "/*" ] ]
+          - Effect: Allow
+            Action: "s3:List*"
+            Resource: !Join [ '', [ !GetAtt MySecondBucket.Arn, "/*" ] ]
+          - Effect: Allow
+            Action: "s3:Get*"
+            Resource: !Join [ '', [ !GetAtt MySecondBucket.Arn, "/*" ] ]
```
Comment (jflans) [inline 03-iam/lab-3.3.2/cfn.yaml:31]: You can combine these into a single stanza

File: 03-iam/README.md [hunk 2]
```diff
@@ -366,11 +373,15 @@ _Is it possible to limit uploads of objects with a specific prefix (e.g.
 starting with "lebowski/") to an S3 bucket using IAM conditions? If not, how else
 could this be accomplished?_
 
+> not that i can tell
+
```
Comment (jflans) [inline 03-iam/README.md:377]: You can do this - see https://aws.amazon.com/blogs/security/writing-iam-policies-grant-access-to-user-specific-folders-in-an-amazon-s3-bucket/ for how

**Review by jflans — APPROVED** [review]
LGTM. We should go over the s3 prefix limiting briefly, but don't let that delay you in your next module

File: 03-iam/README.md (hunk 2 above)
Comment (jdix531) [reply 03-iam/README.md:377]: I did end up finding and implementing that, forgot to update/remove my comment !

**Pull request merged**